phalcon/Annotations/Adapter/Stream.zep

```diff
@@ -20,7 +20,7 @@ use Phalcon\Annotations\Exception;
  * Stores the parsed annotations in files. This adapter is suitable for production
  *
  *<code>
- * use Phalcon\Annotations\Adapter\Files;
+ * use Phalcon\Annotations\Adapter\Stream;
  *
  * $annotations = new Files(
```

Comment:
```suggestion
 * $annotations = new Stream(
```

Reply:
- At the top of class annotation: `Phalcon\Annotations\Adapter\Files` => `Phalcon\Annotations\Adapter\Stream`
- Constructor annotation: `Phalcon\Annotations\Adapter\Files constructor` => `Phalcon\Annotations\Adapter\Stream constructor`

Reply:
Fixed